src/core/DY.Core.Trace.Invariant.fst

```diff
@@ -198,8 +198,8 @@ val event_triggered_at_implies_pred:
   i:timestamp -> prin:principal -> tag:string -> content:bytes ->
   Lemma
   (requires
-    event_triggered_at tr i prin tag content /\
-    trace_invariant tr
+    trace_invariant tr /\
+    event_triggered_at tr i prin tag content
```

Comment:
Okay that makes sense, will revert this change.

Reply:
One could also argue, that having the more interesting (more specific) requirements close to the statement (ensures) of the lemma (i.e. at the end) makes sense.
Especially, in cases with a lot of hypotheses, so that one can see the statement together with the interesting requirements.
Just a thought.

Reply:
I guess we could find many arguments for and against each ordering choice, however I think what matters is to make a consistent choice for the whole codebase, and currently this choice has been from most specific to less specific (except in a few places where this should be fixed). I realize this is missing from the `CONTRIBUTING.md` file, I should document this choice.